client/src/panels/panel_declarations/people/AverageSharePie.js

```diff
@@ -70,6 +72,9 @@ export const AverageSharePie = ({panel_args, sort_func}) => {
                 </span>
               }
             />
+            <div className="centerer">
+              <p><i>*Chart data is averaged over the period listed</i></p>
```

Comment:
Content has to be bilingual. A phrase like this should be stored in yaml and make use of our `text_maker` ecosystem too. Let me know if you haven't been shown that yet!